### ISSUE TITLE
Using `mypy` to verify `check_str_format_call`?

### ISSUE THREAD

**Issue by kornicameister** [kornicameister/loguru-mypy#82]
@ThibaultLemaire Recently I found `mypy.checkexpr.ExpressionChecker.check_str_format_call` method.
I think that is basically what you did as part of #43, right? If that's exactly same thing would you mind replacing your code with the one referenced.

**Comment by ThibaultLemaire** [comment]
> I think that is basically what you did as part of #43, right?

Correct. `visit_call_expr` calls `visit_call_expr_inner` which in turn calls `check_str_format_call`. I even found some of my old breakpoints in there which means I must have been aware of its existence.

I believe I had a good reason not to call it directly, but I can't recall and I don't see anything important in `visit_call_expr_inner` now. Maybe `visit_call_expr` was too big and I was too happy that my code worked to dig any deeper :man_shrugging: 

Well spotted, feel free to change it if you want :+1:

**Comment by kornicameister** [comment]
If I get some time, I will give it a shot.
